src/operators.jl

```diff
@@ -264,6 +264,19 @@ end
 Base.isapprox(p::_APL, α; kwargs...) = isapprox(promote(p, α)...; kwargs...)
 Base.isapprox(α, p::_APL; kwargs...) = isapprox(promote(p, α)...; kwargs...)
 
+MA.operate!(::typeof(-), ::AbstractTermLike) = error("not implemented yet")
+
+MA.operate_to!(::AbstractTermLike, ::typeof(-), ::_APL) = error("not implemented yet")
```

Comment:
`AbstractTermLike` subtypes `_APL`, so we have to have a separate method. Do you want me to do `throw(MethodError(...))` instead of `error("not implemented yet")`?